composer.json

```diff
@@ -27,7 +27,7 @@
         "symfony/http-foundation": "~2.4|~3.0|^4.0",
         "symfony/phpunit-bridge": "~3.3|^4.0",
         "symfony/routing": "~2.3|~3.0|^4.0",
-        "twig/twig":       "~1.16|~2.0"
+        "twig/twig":       "^1.34|^2.4"
```

Comment:
`filter` tag is deprecated since Twig 2.9 (https://twig.symfony.com/doc/2.x/deprecated.html#tags), so this change will only solve the deprecation for lower versions, but introducing a new one for higher versions.

Reply:
You're right, but with such versions we can simply replace `filter` with `apply}

Reply:
Until the constraint is updated to `^2.9`, `apply` tag is forbidden.

Reply:
True. At this point, maybe the best solution is removing any filter/tag and keep spaces? With that, we can also keep older twig versions

Reply:
@garak I don't see many reasons why one would want to bump KnpMenu while at the same time pinning Twig to an old release. Even Symfony's 3.4 LTS branch requires Twig `^1.40|^2.9` now.

https://github.com/symfony/twig-bridge/blob/c5cda7f836ccfa80a84c27aec10aa16591333829/composer.json#L20

I'd do the bump and go for `apply`.